python_modules/dagster/dagster_tests/core_tests/test_logging.py

```diff
@@ -154,6 +156,60 @@ def test_multiline_logging_complex():
     assert captured_results[0].split("\n") == expected_results
 
 
+def test_handler_in_log_manager():
```

Comment:
maybe one more test here would be good. I think it would be valuable to test some other attributes of the handled records other than what gets printed to stdout. You can intercept the actual log records with a pattern similar to what I used [here](https://github.com/dagster-io/dagster/pull/4572/files#diff-6295716d0822c08e8eeb2fdeb9cca45337d2d72a92ed947e4cfe1ac55798310eR165), and then make assertions about different properties of the record. I believe this sort of test would fail in the current iteration (just because "extra" is not passed to the underlying _log() call).

Reply:
Smart way to capture log records, added the test following your format